canopen/sdo/base.py

```diff
@@ -30,7 +31,7 @@ def __init__(
         self,
         rx_cobid: int,
         tx_cobid: int,
-        od: ObjectDictionary,
+        od: objectdictionary.ObjectDictionary,
```

Comment:
I have read somewhere that using modules for reference to type hints should be avoided. But I've tested the following code with 3.8 and it seems to work. So perhaps I'm misremembering.

```py
import canopen
def foo(a: canopen.Node):
    pass
```
```

Reply:
Well if you do find such a recommendation and it has a valid reason, then we can still change that later.  I don't see any reason why we shouldn't do it and as it seems to work (at least mypy doesn't complain here), let's try.